### PR TITLE
fix: supply server with GraphiQL subscriptions endpoint environment variable

### PR DESCRIPTION
**Pull request by darahayes**
This fixes a bug where the data sync server configures itself incorrectly when sitting behind a load balancer.

### CODE REVIEW

**Comment by aliok** [comment]
Is this basically just about using wss?

File: roles/provision-data-sync-apb/tasks/provision-sync-server.yml [hunk 1]
```diff
@@ -38,39 +69,11 @@
         value: "{{ postgres_user}}"
       - name: POSTGRES_DATABASE
         value: "{{ postgres_database }}"
+      - name: GRAPHIQL_SUBS_ENDPOINT
```
Comment (wtrocki) [inline roles/provision-data-sync-apb/tasks/provision-sync-server.yml:72]: SUBS? Could it be `WEBSOCKET`? It could be used for more than subscriptions.

**Comment by wtrocki** [comment]
Great to see this fixed in such sort time! Great work @darahayes 💯 !

Edit:
Closed by mistake. Wrong button 😢

File: roles/provision-data-sync-apb/tasks/provision-sync-server.yml (hunk 1 above)
Comment (darahayes) [reply roles/provision-data-sync-apb/tasks/provision-sync-server.yml:72]: @wtrocki really it's a specific piece of config, and now that https://github.com/aerogear/data-sync-server/pull/50 is merged, they need to match.

**Comment by darahayes** [comment]
@aliok kinda. The way the server used to get the subscriptions endpoint name was as follows:

```
ws://${hostname()}:${port}/subscriptions
```

which might evaluate to something like `ws://10.0.0.2:8000/subscriptions`

This doesn't work if your service sits behind a load balancer (http or https) because the host isn't accessible directly and the ports will usually be mapped also.

In reality it should be something like `wss://my-data-sync-service.nip.io` (HTTPS) or `ws://my-data-sync-service.nip.io` (HTTP). I've defaulted to `wss` in the apb because the apb intends to set the thing up with https however you could manually edit the env variable and restart the pod also if you want to do http (in demo scenarios)

**Review by wtrocki — APPROVED** [review]

**Review by aliok — APPROVED** [review]

**Pull request merged**